README.md

```diff
@@ -1 +1,75 @@
-# Review-code
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+
+
```

Comment:
Great Job so far, but little changes to do you should have to add description in the README file. This will make 
your readme file more professional and anyone that use your Project will have the appropriate information. 😄